src/test/java/api/requests/RequestsAPIRetrievalTests.java

```diff
@@ -162,6 +163,10 @@ void canGetARequestById() {
     assertThat(itemSummary.getString("copyNumber"),
       is(ONE_COPY_NUMBER));
 
+    final JsonObject instanceSummary = representation.getJsonObject("instance");
+    assertThat("title is taken from item",
```

Comment:
```suggestion
    assertThat("title is taken from instance",
```
Please fix in other files as well